### PR TITLE
ToolStatus: Added setting for auto focus to search box.

### PR DESCRIPTION
**Pull request by php940**
Hello @dmex.

This is my first try to contribute to open source project. In this pull request I've tried to make a enhancement for auto focusing search box as suggested in issue #521.

Forgive me if I made a mistake for my first pull request.
Best regards.

### CODE REVIEW

File: plugins/ToolStatus/main.c [hunk 1]
```diff
@@ -1333,6 +1336,9 @@ VOID NTAPI MainWindowShowingCallback(
     {
         SetMenu(PhMainWndHandle, NULL);
     }
+
+    if (ToolStatusConfig.SearchAutoFocus)
```
Comment (php940) [inline plugins/ToolStatus/main.c]: Do I need to check ToolStatusConfig.SearchBoxEnabled here?

File: plugins/ToolStatus/toolstatus.h [hunk 2]
```diff
@@ -115,7 +115,8 @@ typedef union _TOOLSTATUS_CONFIG
         ULONG ResolveGhostWindows : 1;
         ULONG ModernIcons : 1;
         ULONG AutoHideMenu : 1;
-        ULONG Reserved : 4;
+        ULONG SearchAutoFocus : 1;
+        ULONG Reserved : 3;
```
Comment (dmex) [inline plugins/ToolStatus/toolstatus.h]: You'll need to use the Spare bits rather than Reserved since they were used by 2.39 and would conflict.

**Comment by php940** [comment]
Thank you @dmex  for your feedback. 

Hope dc388a9 this fix it.

**Comment by dmex** [comment]
@php940 

Thanks! 

I've approved the PR but I'm on the road and will be a few days before I can test it properly and merge it 👍

**Comment by dmex** [comment]
@php940 

This has been merged. 👍 

Sorry for the delay, we've had some fire evacuations in Australia.

**Pull request merged**